config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :marshal
```

Comment:
Using `:json` should be safe in this app.

Reply:
Switched to `:hybrid`. We can still change later.

Reply:
Who logs in? Only @dbrgn does. So we should be fine with `json`.